cmd/admin-bucket-remote-add.go

```diff
@@ -88,9 +98,10 @@ EXAMPLES:
          --service "replication" --region "us-west-1"
 
   2. Set a new remote replication target 'targetbucket' in region "us-west-1" on https://minio.siteb.example.com for
-     bucket 'sourcebucket' with bandwidth set to 2 gigabits per second.
+	 bucket 'sourcebucket' with bandwidth set to 2 gigabits per second. Enable synchronous replication to the target
+	 and perform health check of target every 100 seconds
      {{.Prompt}} {{.HelpName}} sitea/sourcebucket https://foobar:foo12345@minio.siteb.example.com/targetbucket \
-         --service "replication" --region "us-west-1 --bandwidth "2G"
+         --service "replication" --region "us-west-1 --bandwidth "2G" --sync --healthcheck-seconds 100
```

Comment:
```suggestion
         --service "replication" --region "us-west-1 --bandwidth "2G" --sync
```

We can leave our healthcheck tunable 60secs is more than enough for most, it is present for most advanced use cases only.